### ISSUE TITLE
Allow specifying arbitrary scheduler command line arguments in Q

### ISSUE THREAD

**Issue by klmr** [mschubert/clustermq#28]
Currently the scheduler is set up via a configuration file that’s loaded when the clustermq package is attached.

However, there are situations where additional options need to be specified in individual calls. For instance, I need to specify the `-n` option of the LSF scheduler, and I need to do so on a case-per-case basis.

Of course it would be possible to add this as another configurable option to the scheduler configuration, analogous to `memory`. But maybe a more flexible way would be to allow passing arbitrary arguments to the scheduler; for instance:

```r
clustermq::Q(fun, args, scheduler_args = list('-n4', '"-Rspan[hosts=1]"')
```

**Comment by mschubert** [comment]
I already thought about this and the following options:

* allowing the user specify separate template files with each call of `Q`
* providing function arguments for each option that a user likely wants to change per call; we currently do this for `memory` and `walltime`
* providing a more structured approach where a user can override each template value (`{{ ... }}`) with either an option or a separate argument to `Q`

I'm currently thinking the best option is to retire `walltime` (and `memory`?) in favor of an argument that provides a named list to override arbitrary template values (essentially, what you suggest).

Any thoughts/opinions/objections?

**Comment by mschubert** [comment]
Have a look at [pre-0.6.0](https://github.com/mschubert/clustermq/tree/0.6.0).

Fills any values supplied by `scheduler_args` in the template, but still keeps `memory` as a shorthand for `scheduler_args=list(memory=...)`. Less frequent options like `walltime` are only supported via `scheduler_args`.

**Comment by klmr** [comment]
Great, I’ll check it out as soon as I get the time. Thanks.

**Comment by mschubert** [comment]
solved in `0.5.99`
